infrastructure/ai/prompt_manager.py

```
@@ -22,10 +22,40 @@
+def _get_seed_path() -> Path:
+    """获取种子文件路径，打包后优先从数据目录或 bundle 资源目录读取。
+
+    优先级：
+    1. DATA_DIR/prompts/          — 首次启动时从 bundle 复制过来的位置
+    2. sys._MEIPASS 打包资源       — PyInstaller freeze 时的 bundle 目录
+    3. 源码目录（开发用）           — fallback
+    """
+    import os, sys
+    from application.paths import DATA_DIR
+
+    # 1. 数据目录（生产环境首选）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)